docs/index.md

```diff
@@ -0,0 +1,329 @@
+# lolex
+
+JavaScript implementation of timing related web APIs.
+
+**Implements:**
+
+* [`setTimeout`](#settimeoutcallback-timeout-args)
+* [`clearTimeout`](#cleartimeoutid)
+* [`setImmediate`](#setimmediatecallback-args)
+* [`clearImmediate`](#clearimmediateid)
+* [`setInterval`](#setintervalcallback-timeout-args)
+* [`clearInterval`](#clearintervalid)
+* [`requestAnimationFrame`](#requestanimationframecallback)
+* [`cancelAnimationFrame`](#cancelanimationframeid)
+* [`Date`](#date)
+* [`nextTick`](#nexttickcallback-args)
+* [`queueMicrotask`](#queuemicrotaskcallback)
+* [`performance.now`](#performancenow)
+
+along with a clock instance that controls the flow of time.
+**lolex** also provides a [`Date`](#date) implementation that gets its time from the clock.
+
+**lolex** can be used to simulate passing time in automated tests and other
+situations where you want the scheduling semantics, but don't want to actually
+wait (however, from version 2.0 **lolex** supports those of you who would like to wait too).
+
+**lolex** is a part of [Sinon.JS](https://github.com/sinonjs/sinon.js).
+
+## Installation
+
+```shell
+npm install lolex --save-dev
+```
+
+You are always free to [build it yourself](https://github.com/sinonjs/lolex/blob/master/package.json#L22), of course.
+
+## Usage
+
+**lolex** can be used in both Node.js and browser environments.
+
+In addition in browser environment **lolex** provides a [`performance`](#performancenow) implementation that gets its time from the clock.
+In Node.js environments **lolex** provides a [`nextTick`](#nexttick) implementation that is synchronized with the clock - and a [`process.hrtime`](#hrtimeprevtime) shim that works with the clock.
+
+If you want to use **lolex** in a browser you can use
+[the pre-built version](https://github.com/sinonjs/lolex/blob/master/lolex.js)
+available in the repo and the npm package.
+
+Using npm you only need to reference **lolex** in your `<script>` tags.
+
+```html
+<script type="text/javascript" src="https://unpkg.com/lolex@1.4.0/src/lolex.js">
```

Comment:
Maybe not link to 1.4.0?

Reply:
https://www.cdnpkg.com/lolex/file/lolex.js/
It's the most recent version available.

Reply:
It's not, see https://www.npmjs.com/package/lolex?activeTab=versions

`unpkg` has them all, e.g. https://unpkg.com/lolex@3.0.0/src/lolex-src.js, or just https://unpkg.com/lolex which always redirects to `latest`

Sorta related, we should probably add a `browser` field to package.json so it points to the umd module instead: https://github.com/sinonjs/lolex/blob/master/lolex.js

(or manually point to it instead of `src/lolex`)

Reply:
@SimenB thanks you! I didn't see. I applied this to the docs :+1:

Reply:
Also changed the text to mention `unpkg` instead of `npm`
> Using `unpkg` you only need to reference **lolex** in your `<script>` tags.